homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -318,6 +322,10 @@ async def async_create_miio_device_and_coordinator(
         device = AirPurifier(host, token)
     elif model.startswith("zhimi.airfresh."):
         device = AirFresh(host, token)
+    elif model in MODEL_AIRFRESH_A1:
```

Comment:
`MODEL_AIRFRESH_A1` is a string and not a container. If we want to match strings we should use `==` or `str.startswith` or similar.